SnowflakeTableJournal.cs

```diff
@@ -24,20 +22,9 @@ public SnowflakeTableJournal(Func<IConnectionManager> connectionManager, Func<IU
             FqUnquotedSchemaTableName = $"{SchemaTableSchema}.{UnquotedSchemaTableName}";
         }
 
-        public override void StoreExecutedScript(SqlScript script, Func<IDbCommand> dbCommandFactory)
-        {
-            EnsureTableExistsAndIsLatestVersion(dbCommandFactory);
-            using (var insertScriptCommand = dbCommandFactory())
-            {
-                var timestamp = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds().ToString();
-                insertScriptCommand.CommandText = GetInsertJournalEntrySql(script.Name, timestamp);
-                insertScriptCommand.ExecuteNonQuery();
-            }
-        }
-
         protected override string GetInsertJournalEntrySql(string scriptName, string applied)
         {
-            return $"insert into {FqUnquotedSchemaTableName} (ScriptName, Applied) values ('{scriptName}', '{applied}')";
+            return $"insert into {FqUnquotedSchemaTableName} (ScriptName, Applied) values (?, ?)";
```

Comment:
I haven't tested this, but I believe this will allow the base implementation of `StoreExecutedScript` to work.